From 79c9b60502449b330f155c082b198dfb509c0874 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Wed, 30 Jan 2013 00:02:12 -0500
Subject: [PATCH] libMirage: CMake: don't include GtkDoc and
 GObjectIntrospection unconditionally

Including GtkDoc.cmake and GObjectIntrospection.cmake unconditionally
checks for gtk-doc.pc and gobject-introspection-1.0.pc respectively,
defeating the point of the conditional pkg_check_modules() calls in the
make CMakeLists.txt script.
---
 libmirage/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libmirage/CMakeLists.txt b/libmirage/CMakeLists.txt
index ff72aab..8049a80 100644
--- a/libmirage/CMakeLists.txt
+++ b/libmirage/CMakeLists.txt
@@ -27,8 +27,6 @@ include (CheckCSourceCompiles)
 include (CheckCCompilerFlag)
 include (GNUInstallDirs)
 
-include (GObjectIntrospection)
-include (GtkDoc)
 include (Utilities)
 include (FileList)
 
@@ -50,10 +48,12 @@ pkg_check_modules (LIBLZMA REQUIRED liblzma>=5.0.0)
 pkg_check_modules (SNDFILE REQUIRED sndfile>=1.0.0)
 
 if (INTROSPECTION_ENABLED)
+    include (GObjectIntrospection)
     pkg_check_modules (INTROSPECTION REQUIRED gobject-introspection-1.0>=1.30.0)
 endif ()
 
 if (GTKDOC_ENABLED)
+    include (GtkDoc)
     pkg_check_modules (GTKDOC REQUIRED gtk-doc>=1.4)
 endif ()
 
-- 
1.8.1.1

